data/brands/shop/supermarket.json

```
@@ -30,6 +30,16 @@
     }
   },
   "items": [
+    {
+      "displayName": "Foodtown",
+      "locationSet": {"include": ["us"]},
```

I'd restrict this to only the states they're currently operating in since there's similarly named supermarkets in other parts of the US
```suggestion
      "locationSet": {"include": ["us-nj.geojson", "us-ny.geojson", "us-ct.geojson", "us-pa.geojson"]},
```